.github/workflows/main.yml

```diff
@@ -23,4 +30,5 @@ jobs:
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
           OPENAI_API_MODEL: "gpt-4" # Optional: defaults to "gpt-4"
           exclude: "**/*.json, **/*.md" # Optional: exclude patterns separated by commas
+          review-guidelines: ${{ steps.read-guidelines.outputs.review_guidelines }}
```

Comment:
It seems like you are trying to use the output of a step that is not defined in this workflow file. Make sure to include the step that sets `steps.read-guidelines.outputs.review_guidelines` before this line.